docs/project_droneworld/about.md

```diff
@@ -1,58 +1,61 @@
+Drone World, developed by Dr. Ankit Agrawal and his team, is an advanced simulation platform for testing small unmanned aerial systems (sUAS). It enables users to configure detailed test scenarios by specifying environmental conditions (e.g., weather, terrain), sUAS capabilities (e.g., sensors, hardware), and mission objectives. The platform generates a realistic 3D simulation environment and monitors data to ensure safety and detect issues, producing comprehensive test reports with detailed analysis.
+
+By automating and streamlining the testing process, Drone World improves safety, reliability, and efficiency for drone developers. It allows comprehensive pre-flight testing in ultra-realistic environments, helping developers refine their systems and iterate more rapidly on complex missions. Our team at OSS is dedicated to enhancing Drone World’s capabilities, including refining environmental settings, drone configurations, and integrating new features.
 
 ### Information
 
 - **Source Code:**  https://github.com/oss-slu/DroneWorld [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/DroneWorld)
-- **Client** [Dr.Ankit Agrawal](https://www.slu.edu/science-and-engineering/academics/computer-science/faculty-and-staff/ankit-agrawal.php) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ankyAgrawal)
-- **Current Tech Lead:** [Yashaswini Shivalingaiah](https://yashaswinishivalingaiah.com/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashaswini-slu)  [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashaswini-shivalingaiah-467a9652/)
+- **Client:** [Dr. Ankit Agrawal](https://www.slu.edu/science-and-engineering/academics/computer-science/faculty-and-staff/ankit-agrawal.php) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ankyAgrawal)
+- **Current Tech Lead:** Mohammad Al-Hanoosh [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/mohamdlog)
 - **Developers:**
-    - Kaleb Yu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/kalyus)
-    - Szymon Rostkowski [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/sr259)
-    - Hebron Bekele [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/hebronh) 
+    <!--[<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/)-->
```

Comment:
avoid comment cruft whenever possible. overall, you did a lot and made some great improvements!